### DISCUSSION TITLE
[Script request] Immich

### DISCUSSION THREAD

**Discussion by sbminer123** [community-scripts/ProxmoxVE#713] [Request script]
### Application Name

Immich

### Website

https://immich.app/

### Description

Immich is a self hosted Photo Hosting platform, seems theres a docker image available but I would like to create a Lxc container for it 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Narkowii** [comment]
Use Dockge LXC script on this site and choose option Install Immich

**Reply by sbminer123** [reply]
Awesome, Thank you. Still would be cool to have its own Lxc container tho, I dont like the way file permissions work in Docker containers but I have learned the way it works with LXC containers.

If you have any resources for me to learn about docker containers and how the file systems work that would be helpful 😁

**Reply by sendmebits** [reply]
Could the Dockge script be updated so that when doing a script search for "Immich" it gets a hit on it? 

I ran into this myself struggling to search for Immich even though I knew it was there somewhere.
